worker/raft/raftclusterer/worker_test.go

```diff
@@ -286,3 +253,15 @@ func (s *WorkerSuite) publishDetails(c *gc.C, details apiserver.Details) {
 		c.Fatal("timed out waiting for details to be received")
 	}
 }
+
+// Connect  all of the provided transport bidirectionally.
+func connectTransports(transports []raft.LoopbackTransport) {
```

Comment:
I think if you change this to:
func connectTransports(transports []raft.LoopbackTransport...)

you can then call it with
connectTransports(t1, t2, t3)

and not have to cast them into a slice in the test code.